README.md

```diff
@@ -635,6 +635,43 @@ Translations of the guide are available in the following languages:
    end
    ```
 
+* <a name="parallel-assignment"></a>
+    Avoid the use of parallel assignment for defining variables, except for
+    the return of method calls, use with the splat operator, or variable
+    reassignment. Parallel assignment is less readable and slower.
+<sup>[[link](#parallel-assignment)]</sup>
+
+  ```Ruby
+  # bad
+  a, b, c, d = 'foo', 'bar', 'baz', 'foobar'
+
+  # good
+  a = 'foo'
+  b = 'bar'
+  c = 'baz'
+  d = 'foobar'
+
+  # good - variable reassignment
```

Comment:
I agree, having some extra information about variable reassignment would be good. I added a sub section on variable reassignment. I think that it may need to be rewritten though.